pkg/core/state/nep17.go

```diff
@@ -44,10 +44,15 @@ type NEP17Transfer struct {
 	Tx util.Uint256
 }
 
-// NEP17TransferInfo is a map of the NEP17 contract IDs
-// to the corresponding structures.
+// NEP17Balances is a map of the NEP17 contract IDs to the corresponding structures.
+type NEP17Balances struct {
+	Trackers map[int32]NEP17Tracker
+}
+
+// NEP17TransferInfo stores map of the NEP17 contract IDs to the balance's last updated
+// block trackers along with information about NEP17 transfer batch.
 type NEP17TransferInfo struct {
-	LastUpdated map[int32]NEP17Tracker
+	LastUpdated map[int32]uint32
```

Comment:
The previous commit (renaming this field) is connected to this, I would rather squash them.

Reply:
This part is still relevant, I think. Otherwise it's not clear why we're renaming things.